L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp3.cc

```diff
@@ -654,14 +654,15 @@ HGCalTriggerGeometryBase::geom_set HGCalTriggerGeometryV9Imp3::getStage1LinksFro
   return stage1link_ids;
 }
 
-HGCalTriggerGeometryBase::geom_set HGCalTriggerGeometryV9Imp3::getLpgbtsFromStage1Fpga(const unsigned stage1_id) const {
-  geom_set lpgbt_ids;
+std::vector<unsigned> HGCalTriggerGeometryV9Imp3::getLpgbtsFromStage1Fpga(const unsigned stage1_id) const {
+  std::vector<unsigned> lpgbt_ids;
   HGCalTriggerBackendDetId id(stage1_id);
 
-  auto stage1_itrs = stage1_to_lpgbts_.equal_range(id.label());
-  for (auto stage1_itr = stage1_itrs.first; stage1_itr != stage1_itrs.second; stage1_itr++) {
-    lpgbt_ids.emplace(HGCalTriggerBackendDetId(
-        id.zside(), HGCalTriggerBackendDetId::BackendType::LpGBT, id.sector(), stage1_itr->second));
+  auto stage1_itrs = stage1_to_lpgbts_.at(id.label());
```

Comment:
- Probably better to explicitly use `const`, even though `auto`might be able to infer it automatically given that the function is `const`
- Better to change the variable name since `at()` is not returning iterators, but reference to the value accessed by the key